.github/workflows/release-repo-dispatch.yml

```diff
@@ -0,0 +1,18 @@
+name: release-repo-dispatch
+
+on:
+  repository_dispatch:
+
+jobs:
+  update_version_from_repository_dispatch:
+    uses: killbill/gh-actions-shared/.github/workflows/update-version.yml@main
+
+  # If CI failed, should we revert commit? I don't think so, since we still have release.yml with "workflow_dispatch",
+  # fix broken things and then trigger release.yml manually.
+  run-ci:
+    uses: ./.github/workflows/ci.yml
+    needs: update_version_from_repository_dispatch
+
+  release:
```

Comment:
To your comment (https://github.com/killbill/gh-actions-shared/pull/41/files#r1232894706), I'm thinking indeed that we don't need to share that workflow.

But I also think we don't even need it. You could just call `release.yml` and specify the new version to update: https://github.com/killbill/killbill-oss-parent/blob/24ce1d6a1c426086d072e65b72b4a64810292b2a/.github/workflows/release.yml#L7

Reply:
> .... You could just call release.yml and specify the new version to update:

@pierre do you mean as something like:

```jsonp
....

    release-jobs:
      name: "Test Various Workflow"
      uses: ./.github/workflows/dispatch_and_call.yml
      with:
        commons_version: <from_repository_dispatch>
....
```
? If so, it is not working. See this [action result](https://github.com/xsalefter/kb-oss-parent/actions/runs/5458456689/jobs/9933564140) (`hello_msg` and `is_exist` is not set at all). This is repeatable and you can try [here](https://github.com/xsalefter/kb-oss-parent/actions/workflows/test.yml). main workflow file that simulate our `release.yml` is [here](https://github.com/xsalefter/kb-oss-parent/actions/workflows/dispatch_and_call.yml).

Reply:
I think we don't need the shared workflow (nor this PR).

In https://github.com/killbill/killbill-commons/pull/167/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R118, could you call the workflow dispatch to trigger the existing `release.yml` workflow (and specify `commons_version` in the payload)?

Reply:
@pierre I update PR in commons. The payload now:

```bash
"{\"event_type\":\"update_kb_commons_version\", \"client_payload\": {\"commons_version\": \"$PROJECT_VERSION\"}}"
```

you can give a feedback. 

As PR in this repo, I think we need at least:

```yml
repository_dispatch:
    types: [update_kb_commons_version, update_kb_base_plugin_version, update_kb_platform_version, update_kb_client_version]
```

as you mentioned [above](https://github.com/killbill/killbill-oss-parent/pull/755#discussion_r1235214301), no?